tools/add_triage_signature.py

```diff
@@ -2149,6 +2124,53 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket("\n".join(host_messages))
 
 
+class UserManagedNetworkingLoadBalancer(ErrorSignature):
+    lb_operators = ['authentication', 'console', 'ingress']
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            function_impact_label="umh_load_balancer_unhealthy",
+            comment_identifying_string="h1. Probably user managed load-balancer issues, nothing we can do",
+        )
+
+    def operator_unhealthy(self, operator):
+        check_conditions = [('Available', False), ('Progressing', True), ('Degraded', True)]
+        for condition, result in check_conditions:
+            if operator[condition]['result'] is result:
+                return True
+        return False
+
+    def _process_ticket(self, url, issue_key):
+        md = get_metadata_json(url)
+        cluster_md = md["cluster"]
+        if cluster_md["user_managed_networking"] is not True and cluster_md["user_managed_networking"] != "true":
+            return
+
+        try:
+            controller_logs = get_triage_logs_tar(
+                triage_url=url, cluster_id=get_metadata_json(url)["cluster"]["id"]
+            ).get("controller_logs.tar.gz/assisted-installer-controller-*.logs")
+        except FileNotFoundError:
+            return
+
+        statuses = operator_statuses_from_controller_logs(controller_logs)
+        unhealthy_operators = [operator for operator in statuses if self.operator_unhealthy(statuses[operator])]
+
+        if self.lb_operators.sort() == unhealthy_operators.sort():
```

Comment:
Compare Python [sets](https://realpython.com/python-sets/) rather than lists, this way you can avoid sort() as order doesn't matter